src/CMakeLists.txt

```diff
@@ -231,11 +230,14 @@ set(GAMEENGINE_SRC
     game/ui/msgbox.cpp
 )
 
-# Glob all the header files together to add to the project for msvc/xcode.
```

Comment:
You need to do the glob still and then do another "foreach" loop to create the "Header Files" folder to mirror the "Source Files" folder.